plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -894,10 +895,14 @@ public void testCreatePartitionedTableWithNestedTypes()
     @Test
     public void testCreatePartitionedTableAs()
     {
+        File tempDir = createTempDir();
+        String locationPath = tempDir.toURI().toASCIIString();
         assertUpdate(
                 "CREATE TABLE test_create_partitioned_table_as " +
                         "WITH (" +
                         "partitioning = ARRAY['ORDER_STATUS', 'Ship_Priority', 'Bucket(order_key,9)']" +
+                        ", " +
+                        "location = '" + locationPath + "'" +
```

Comment:
add location first (as it's "more general")

```suggestion
                        "location = '" + locationPath + "', " +
                        "partitioning = ARRAY['ORDER_STATUS', 'Ship_Priority', 'Bucket(order_key,9)']" +
```